.github/workflows/commit-format-check.yaml

```diff
@@ -0,0 +1,70 @@
+name: Commits Formatting Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [staging]
+
+jobs:
+  commit:
+    name: Commit and PR Format Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Commit Lines Length Check
+        if: ${{ github.actor != 'dependabot' && github.actor != 'dependabot[bot]' }}
```

Comment:
In the future we might have dependabot on to auto-bump actions, that's why this check is added.

Dependabot does not fully respect these standards.